pom.xml

```diff
@@ -123,7 +129,18 @@
 			<version>${secure-properties.version}</version>
 			<classifier>mule-plugin</classifier>
 		</dependency>
-	</dependencies>
+		<dependency>
+            <groupId>com.mulesoft.connectors</groupId>
+            <artifactId>mule-mongodb-connector</artifactId>
+            <version>6.3.0</version>
```

Comment:
Please externalize this version in the <properties> section of the pom.xml

Reply:
Done.